sdk/identity/azure-identity/src/main/java/com/azure/identity/InteractiveBrowserCredentialBuilder.java

```diff
@@ -89,9 +109,8 @@ InteractiveBrowserCredentialBuilder disableAutomaticAuthentication() {
     public InteractiveBrowserCredential build() {
         ValidationUtil.validate(getClass().getSimpleName(), new HashMap<String, Object>() {{
                 put("clientId", clientId);
-                put("port", port);
             }});
-        return new InteractiveBrowserCredential(clientId, tenantId, port, automaticAuthentication,
+        return new InteractiveBrowserCredential(clientId, tenantId, port, redirectURL, automaticAuthentication,
```

Comment:
What if the `redirectUrl` contains a port already? Should we throw here is both `port` and `redirectUrl` are specified?

Reply:
As discussed, to fail early with a better error message, added the check here to throw if both port and redirectUrl are specified.